inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.hpp

```diff
@@ -18,43 +18,77 @@
 
 using TransformationTests = CommonTestUtils::TestsCommon;
 
-std::pair<bool, std::string> compare_functions(
+class FunctionsComparator {
+public:
+    enum CmpValues {
+        NONE = 0,
+        CONST_VALUES = 1 << 0,
+        NAMES = 1 << 1,
+        RUNTIME_KEYS = 1 << 2,
+        PRECISIONS = 1 << 3,
+        ATTRIBUTES = 1 << 4,
+    };
+
+    struct Result {
+        bool valid;
+        std::string message;
+    };
+
+    static constexpr FunctionsComparator no_default() noexcept {
```

Comment:
What do names `no_default` and `with_default` means?

Reply:
`with_default` cover defaults from `compare_function` and for `no_default` all conditional check are disabled.